lib/network/encoding.go

```diff
@@ -192,6 +192,7 @@ func MarshalRegisteredType(data ProtocolMessage) ([]byte, error) {
 	var err error
 	if buf, err = protobuf.Encode(data); err != nil {
 		dbg.Error("Error for protobuf encoding:", err)
+		dbg.Error(dbg.Stack())
```

Comment:
In fact that's probably something we need to comment. I was not sure whether to leave it in or not.